instagram-app/src/components/PostContainer/PostsPage.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import dummyData from '../../dummy-data'
+import SearchBar from '../SearchBar/SearchBar'
+import PostContainer from './PostContainer';
+
+class PostsPage extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      posts: [],
+      filteredPosts: [],
+      searchTerm : 'philz'
+    }
+  }
+
+  componentDidMount() {
+    this.setState({
+      posts: dummyData
+    });
+  }
+
+  searchPostsHandler = e => {
+    const posts = this.state.posts.filter(post => {
+      if (post.username.includes(e.target.value)) {
```

Comment:
great work filtering the posts. But to not have to store the posts twice. You can just filter them before you pass them down. Instead of setting state with the filtered posts.